packages/core/src/mesh/SkinnedMeshRenderer.ts

```diff
@@ -244,8 +253,25 @@ export class SkinnedMeshRenderer extends MeshRenderer {
   /**
    * @internal
    */
-  override _cloneTo(target: SkinnedMeshRenderer): void {
-    super._cloneTo(target);
+  override _cloneTo(target: SkinnedMeshRenderer, srcRoot: Entity, targetRoot: Entity): void {
```

Comment:
if(this._jointEntities) ->  if(this._bones)

Reply:
done